### PR TITLE
Add deposit dates page styles

### PR DESCRIPTION
**Pull request by viktor-yakubiv**
@Joozty this would work. It breaks other pages look because we need use Main or Card instead Fragment. But we can fix it later when necessary.

### CODE REVIEW

**Comment by Joozty** [comment]
It looks ugly on page load. Is it related to the stuff you mentioned in the PR description?

https://www.loom.com/share/a8f382767ec04e00976fa20bf757085c

**Comment by viktor-yakubiv** [comment]
Yes, it does. I thought it only on my computer something. I do not why it distributes the space if containers are small.

I will take a look at it.

File: pages/deposit-dates.css [hunk 1]
```diff
@@ -1,46 +1,47 @@
...
-.row {
+.container {
   display: flex;
-  flex-wrap: nowrap;
-  margin-bottom: 2rem;
-
-  @media (--screen-until-desktop) {
-    flex-wrap: wrap;
+  flex-wrap: wrap;
+  align-content: flex-start;
```
Comment (viktor-yakubiv) [inline pages/deposit-dates.css]: This line fixes [the bug](https://github.com/oacore/dashboard/pull/41#issuecomment-568205794).

File: pages/deposit-dates.css [hunk 2]
```diff
@@ -1,46 +1,47 @@
-/* TODO: Custom media from layout.css doesn't work */
-@custom-media --screen-until-desktop (max-width: 727px);
-
-.table thead > :first-child {
-  border-top-width: 2px;
+%item {
+  flex: 3 3 90%;
+  margin: 0 var(--component-spacer-x, 1rem) var(--component-spacer-y, 1.5rem);
```
Comment (Joozty) [inline pages/deposit-dates.css]: Does this variable even exist? Can you please always explicitly add TODO comment when something should be created.

**Pull request merged**